djlsp/scripts/django-collector.py

```diff
@@ -390,6 +391,22 @@ def collect_project_data():
     }
 
 
+def get_default_django_settings_module():
+    try:
+        # Patch django execute to prevent it from running when calling main
+        with patch(
+            "django.core.management.execute_from_command_line", return_value=None
+        ):
+            from manage import main
+
+            # Need to run main becuase default env is set in main function
+            main()
+
+        return os.getenv("DJANGO_SETTINGS_MODULE")
+    except ImportError:
+        return ""
```

Comment:
Might want to log or return that settings module hasn't been found

Reply:
There is no logging facility for the django-collector script yet.